app/controllers/idv/hybrid_mobile/document_capture_controller.rb

```diff
@@ -3,6 +3,7 @@ module HybridMobile
     class DocumentCaptureController < ApplicationController
       include DocumentCaptureConcern
       include HybridMobileConcern
+      include PhoneQuestionAbTestConcern
```

Comment:
Charley and I discussed that this include isn't strictly necessary since it's included via HybridMobileConcern, but it's 1) good for transparency and 2) temporary while the A/B test is in process. Same above in the other DocumentCaptureController and IdvStepConcern.